packages/babel-node/scripts/list-node-flags.js

```diff
@@ -0,0 +1,38 @@
+// NOTE: This file must be runnable on all Node.js version
+
+const cp = require("child_process");
+const fs = require("fs");
+const path = require("path");
+
+const flags = require("../data/node-flags-with-value.json");
+const knownFlagsSet = new Set(flags);
+
+function push(flag) {
+  if (knownFlagsSet.has(flag)) return;
+
+  if (!process.argv.includes("--update")) {
+    console.log(
+      `Missing flag "${flag}" from Node.js ${process.version} in ./packages/babel-node/data/node-flags-with-value.json.\n` +
+        `Run node ./packages/babel-node/scripts/list-node-flags.js --update to update it.`
+    );
+    // eslint-disable-next-line n/no-process-exit
+    process.exit(1);
```

Comment:
Nit: we can just throw `Missing flag` as an error so that the process will exit with non-zero code.